source/documentation/rdfstar/__index.md

```diff
@@ -70,6 +67,17 @@ is a prefix for `<http://jena.apache.org/ARQ/function#>`.
 | `afn:triple(?s, ?p, ?o)` | Create a triple term from s/p/o                    |
 | `afn:isTriple(?t)`       | Return true if the argument value is a triple term |
 
+### SPARQL Property Functions related to triple terms
+
+`apf:` is a prefix for `<http://jena.apache.org/ARQ/property#>`.
+
+| Property Function | Description |
+| -------- | ----------- |
+| `<< s p o >> apf:find t` . | Match the triple term. Any `s`, `p`, `o`, and `t` can be RDF terms or variables.|
+
+`apf:find` will result in all the variables being are set accorind to the match,
```

Comment:
will result in all the variables being are set accorind to the match

will result in all the variables being set according to the match

?